### PR TITLE
optimize link description of barprop

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

**Comment by fuchunhui** [comment]
https://github.com/mdn/content/pull/34068#issuecomment-2163552827

File: files/en-us/web/api/barprop/visible/index.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ The **`visible`** read-only property of the {{domxref("BarProp")}} interface ret
 
 ## Value
 
-A {{jsxref("Boolean")}}, which is true if the top-level window is opened by
-{{domxref("window.open")}} with the {{domxref("window.open", "requesting a popup window", "popup", 1)}}.
+A {{jsxref("Boolean")}}, which is true if the top-level window is opened by {{domxref("window.open")}} with the {{domxref("window.open", "popup", "", 1)}} feature enabled.
```
Comment (wbamberg) [inline files/en-us/web/api/barprop/visible/index.md:15]:
```suggestion
A {{jsxref("Boolean")}}, which is true if the top-level window is opened by {{domxref("window.open")}} with the {{domxref("window.open", "popup", "popup")}} feature enabled.
```

**Comment by fuchunhui** [comment]
thanks, where can I see API documentation about `domxref`?

File: files/en-us/web/api/barprop/visible/index.md [hunk 2]
```diff
@@ -12,8 +12,7 @@ The **`visible`** read-only property of the {{domxref("BarProp")}} interface ret
 
 ## Value
 
-A {{jsxref("Boolean")}}, which is true if the top-level window is opened by
-{{domxref("window.open")}} with the {{domxref("window.open", "requesting a popup window", "popup", 1)}}.
+A {{jsxref("Boolean")}}, which is true if the top-level window is opened by {{domxref("window.open")}} with the {{domxref("window.open", "popup", "popup")}} feature enabled.
```
Comment (wbamberg) [inline files/en-us/web/api/barprop/visible/index.md:15]:
```suggestion
A {{jsxref("Boolean")}}, which is true if the top-level window is opened by {{domxref("window.open")}} with the [`popup`](/en-US/docs/Web/API/Window/open#popup) feature enabled.
```

**Comment by wbamberg** [comment]
> thanks, where can I see API documentation about `domxref`?

The only reliable documentation is in the source: https://github.com/mdn/yari/blob/main/kumascript/macros/DOMxRef.ejs. But it seems in this case we can't get it to do what we want, so let's just make it a Markdown link 🤷 .

Thanks for your PR, @fuchunhui !

**Review by wbamberg — APPROVED** [review]
👍

**Pull request merged**